doc/source/dev/contributor/contributor_toc.rst

```diff
@@ -18,44 +23,41 @@ This guide is designed to help you quickly find the information you need about S
 
 Development environment
 -----------------------
-- :ref:`quickerstart-conda` contains just the commands you need to get started on Mac and Linux
-- :ref:`quickstart-pip` presents an overview of setting up the development environment using pip on Linux
-- :ref:`quickstart-mac` presents a step-by-step process for setting up a convenient SciPy development environment in macOS
-- :ref:`quickstart-ubuntu` presents a step-by-step process for setting up a convenient SciPy development environment in Ubuntu
-- :ref:`build-windows` presents a step-by-step process for building SciPy on Windows
+
+- :ref:`system-level` shows how to install system-level dependencies for Linux, Mac or Windows.
+- :ref:`conda-guide` presents a step-by-step process for setting up a convenient SciPy development environment with conda *(recommended)*.
```

Comment:
If this is *recommended*, shouldn't it be first on the list?

Reply:
Good point - let me fix that